scripts/footprint/size_report

```diff
@@ -691,6 +713,8 @@ def parse_args():
                         help="Zephyr base path")
     parser.add_argument("-o", "--output", required=True,
                         help="Output path")
+    parser.add_argument("-w", "--westtopdir", default=None,
+                        help="West path (WEST_TOPDIR)")
```

Comment:
This size report script itself is independent of `west`, so I propose to make it a generic `--workspace` argument.

This would indicate that it is also useful for non-west users, but where code is still located in a common folder / workspace.

```suggestion
    parser.add_argument("-w", "--workspace", default=None,
                        help="Workspace path")
```

Reply:
makes sense